data/autoyast_qam/12_installation.xml.ep

```diff
@@ -266,14 +274,15 @@
     </drive>
   </partitioning>
   % }
-  % unless ($check_var->('ARCH', 'aarch64')) {
+  % if ($check_var->('ARCH', 'x86_64')) {
```

Comment:
Before there is efi partitioning `if ($check_var->('ARCH', 'aarch64'))` and everything else, why do you change everything else to x86_64 ?

Reply:
Good question, for the `disk partitioning`, I would say there might be some wrong configuration before, 

We have dedicate partition setting for `s390x` and `aarch64` already, `unless` logic will count `s390x` in again.

Reply:
ok